### DISCUSSION TITLE
同步功能能否设置要同步的文件类型

### DISCUSSION THREAD

**Discussion by Eason0210** [rime/home#786] [Q&A]
目前的同步功能会将个人配置目录下的所有方案和个人词库同步到sync文件夹，但在机器比较多的时候，会占用较大的磁盘空间。
请问能否实现同步的时候，只同步个人词库文件（比如 `luna_pinyin.userdb`）和相应的`installation.yaml`？

squirrel 0.15.2  or Weasel 0.14.3

**Comment by PaperPrototype** [comment]
Yeah no kidding
